tests/App/Kernel.php

```diff
@@ -63,6 +63,7 @@ protected function configureContainer(ContainerBuilder $container, LoaderInterfa
         $container->loadFromExtension('doctrine', [
             'dbal' => ['url' => 'sqlite://:memory:'],
             'orm' => [
+                'report_fields_where_declared' => true,
```

Comment:
needed on ORM 3. Otherwise:

```
1) Sonata\Doctrine\Tests\Mapper\ORM\DoctrineORMMapperTest::testDoctrineMappingLoaded
InvalidArgumentException: The $reportFieldsWhereDeclared argument is no longer supported, make sure to omit it when calling Doctrine\ORM\Mapping\Driver\AttributeDriver::__construct.

/var/www/sonata-doctrine-extensions/vendor/doctrine/orm/lib/Doctrine/ORM/Mapping/Driver/AttributeDriver.php:45
/tmp/sonata-doctrine-extensions/var/cache/ContainerKqO7EY2/get_Doctrine_Orm_DefaultMetadataDriverService.php:21
/tmp/sonata-doctrine-extensions/var/cache/ContainerKqO7EY2/Sonata_Doctrine_Tests_App_KernelTestDebugContainer.php:98
/tmp/sonata-doctrine-extensions/var/cache/ContainerKqO7EY2/getDoctrine_Orm_DefaultConfigurationService.php:26
/tmp/sonata-doctrine-extensions/var/cache/ContainerKqO7EY2/Sonata_Doctrine_Tests_App_KernelTestDebugContainer.php:98
/tmp/sonata-doctrine-extensions/var/cache/ContainerKqO7EY2/getDoctrine_Orm_DefaultEntityManagerService.php:24
/tmp/sonata-doctrine-extensions/var/cache/ContainerKqO7EY2/getDoctrine_Orm_DefaultEntityManagerService.php:21
/var/www/sonata-doctrine-extensions/vendor/symfony/var-exporter/Internal/LazyObjectState.php:97
/var/www/sonata-doctrine-extensions/vendor/symfony/var-exporter/LazyGhostTrait.php:176
/var/www/sonata-doctrine-extensions/vendor/doctrine/orm/lib/Doctrine/ORM/EntityManager.php:513
/var/www/sonata-doctrine-extensions/vendor/symfony/doctrine-bridge/CacheWarmer/ProxyCacheWarmer.php:48
/var/www/sonata-doctrine-extensions/vendor/symfony/http-kernel/CacheWarmer/CacheWarmerAggregate.php:108
/var/www/sonata-doctrine-extensions/vendor/symfony/http-kernel/Kernel.php:545
/var/www/sonata-doctrine-extensions/vendor/symfony/http-kernel/Kernel.php:763
/var/www/sonata-doctrine-extensions/vendor/symfony/http-kernel/Kernel.php:126
/var/www/sonata-doctrine-extensions/vendor/symfony/framework-bundle/Test/KernelTestCase.php:73
/var/www/sonata-doctrine-extensions/tests/Mapper/ORM/DoctrineORMMapperTest.php:58
/var/www/sonata-doctrine-extensions/vendor/phpunit/phpunit/src/Framework/TestResult.php:728
/var/www/sonata-doctrine-extensions/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/sonata-doctrine-extensions/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/sonata-doctrine-extensions/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/sonata-doctrine-extensions/vendor/phpunit/phpunit/src/TextUI/TestRunner.php:651